### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by DmytroBondariev**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,39 @@
...
+        self.health = health
+        self.hidden = hidden
+        Animal.alive.append(self)
+
+    def __repr__(self) -> str:
+        return "{"\
+               f"Name: {self.name}, " \
+               f"Health: {self.health}, " \
+               f"Hidden: {self.hidden}"\
+               "}"
```
Comment (Vasyl-Smutok) [inline app/main.py:23]: better to use `()` then `\`

File: app/main.py [hunk 2]
```diff
@@ -1 +1,39 @@
-# write your code here
+from __future__ import annotations
```
Comment (Vasyl-Smutok) [inline app/main.py:1]: Unnecessary import

File: app/main.py [hunk 3]
```diff
@@ -1 +1,39 @@
...
+               f"Hidden: {self.hidden}"\
+               "}"
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        if self.hidden is False:
+            self.hidden = True
+        else:
+            self.hidden = False
```
Comment (Vasyl-Smutok) [inline app/main.py:31]: try to write it in one line 

HINT:
use `not`

**Review by Vasyl-Smutok — CHANGES_REQUESTED** [review]

File: app/main.py [hunk 4]
```diff
@@ -1 +1,35 @@
...
+            f"Name: {self.name}, "
+            f"Health: {self.health}, "
+            f"Hidden: {self.hidden}"
+            "}"
+        )
+
+
+class Herbivore(Animal):
+    def hide(self) -> None:
+        self.hidden = True if self.hidden is not True else False
```
Comment (Vasyl-Smutok) [inline app/main.py:27]:
```suggestion
        self.hidden = not self.hidden
```

**Review by Vasyl-Smutok — CHANGES_REQUESTED** [review]

File: app/main.py (hunk 4 above)
Comment (Dmytry95) [reply app/main.py]: а это интересно)

**Review by katevasilenko — APPROVED** [review]
Looks good to me

**Pull request closed without merging**